extensions/wrapper/wrapper-common-api/src/main/java/de/sovity/edc/ext/wrapper/api/common/model/UiAsset.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.common.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+import java.util.Map;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@RequiredArgsConstructor
+@Schema(description = "Asset Details")
+public class UiAsset {
+
+    @Schema(description = "Asset Name", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String name;
+
+    @Schema(description = "Asset Title", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String title;
+
+    @Schema(description = "Asset Language", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String language;
+
+    @Schema(description = "Asset Description", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String description;
+
+    @Schema(description = "Asset Creator", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String creator;
+
+    @Schema(description = "Asset Publisher", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String publisher;
+
+    @Schema(description = "License URL", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String license;
```

Comment:
the differentiation from the documentation should also be a part of the variable name. When somebody uses the TS Client, the documentation doesn't necessarily have to be available well, so I'd recommend a"licenseUrl"